conda/resolve.py

```diff
@@ -409,14 +424,33 @@ def build_conflict_map(self, specs, specs_to_add=None, history_specs=None):
             # records_per_name is a completely arbitrary number here.  It is meant to gather more
             # than just one record, to explore the space of dependencies a bit.  Doing all of them
             #    can be an enormous problem, though.  This is hopefully a good compromise.
-            records_per_name = 7
```

Comment:
removing this limit may slow things down a lot.  Have you done any kind of speed test?  Especially with the anaconda metapackage - the more packages involved, the slower this will go.

Reply:
Tried this out with anaconda metapackage, yep slows it down a ton!